src/promptflow-tracing/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # promptflow-tracing package
 
+## v1.16.1 (2024.12.14)
```

Comment:
Please change this to 1.16.3 as that’s the latest version.

Reply:
Updated to 1.16.13 in changelog